code/game/objects/structures/tank_dispenser.dm

```diff
@@ -34,6 +35,22 @@
 		if(5 to TANK_DISPENSER_CAPACITY)
 			. += "plasma-5"
 
+/obj/structure/tank_dispenser/attack_hand(mob/living/user, list/modifiers)
+	. = ..()
+	if (plasmatanks)
+		dispense(/obj/item/tank/internals/plasma, user)
+		plasmatanks--
+		update_appearance()
+		return
```

Comment:
also this return is redundant, did you mean to put something here?

Reply:
It will continue the attack chain if I won't return.

Reply:
so this is `return NONE`? can you do that explicitly please?

Reply:
This doesn't make any sense, `return` hands back . by default, just the same way putting nothing at the end of a proc does
![image](https://github.com/tgstation/tgstation/assets/58055496/3c9814e8-d668-454c-b07c-7e6d8a6eb84a)